File: parsers/src/main/java/wcl/gpromanager/parsers/tracks/entities/Track.java
package wcl.gpromanager.parsers.tracks.entities;

import lombok.Value;
import lombok.experimental.Accessors;

@Value
@Accessors(fluent=true)
public class Track {
    private int id;
    private String trackName;
    private String imageFileName;
    private String location;
    private float raceDistance;
    private int laps;
    private float lapDistance;
    private float averageSpeed;
    private int corners;
    private float pitlaneTime;
    private int power;
    private int handling;
    private int acceleration;
    private String downforce;
    private String overtaking;
    private String suspensionRigidity;
    private String fuelConsumption;
    private String tyreWear;
    private String gripLevel;
}

File: parsers/src/main/kotlin/wcl/gpromanager/parsers/tracks/entities/Track.kt
package wcl.gpromanager.parsers.tracks.entities

data class Track(
    val id: Int,
    val trackName: String,
    val imageFileName: String,
    val location: String,
    val raceDistance: Float,
    val laps: Int,
    val lapDistance: Float,
    val averageSpeed: Float,
    val corners: Int,
    val pitlaneTime: Float,
    val power: Int,
    val handling: Int,
    val acceleration: Int,
    val downforce: String,
    val overtaking: String,
    val suspensionRigidity: String,
    val fuelConsumption: String,
    val tyreWear: String,
    val gripLevel: String
)

